### DISCUSSION TITLE
Can't delete language

### DISCUSSION THREAD

**Discussion by Nick0w7** [HugoFara/lwt#197] [Q&A] [answered]
When trying to delete a language,  there's no visible button on the page. 
![Screenshot_20240405-200059_Chrome](https://github.com/HugoFara/lwt/assets/164795039/0510cc3b-d371-446e-90b6-8412697f970c)

**Comment by HugoFara** [comment, marked as answer]
You need to delete all texts in the language first, then the button will appear :smile: 

Your comment makes me think that it will be more clear to have a disabled button with explanations instead.
